ARC/main/actions.py

```diff
@@ -1,55 +1,84 @@
 import re
 from main.models import CDC, CourseSlot, Output
-from main.views import map_options
+# from main.views import map_options
 
 from django.shortcuts import render
 from django.http import HttpResponse
 from django.template.response import TemplateResponse
-from main.forms import MapsForm
+from main.forms import MapsForm, SingleCDCForm
 
 
 def single_option_CDC(modeladmin, request, queryset):
```

Comment:
I feel you should be using one convention throughout the code. `forms.py` uses TitleCase and `actions.py` uses case_with_underscore.

Reply:
Okay, will make those changes